.github/scripts/status_update_common.py

```
@@ -0,0 +1,212 @@
+              }
+              fieldValueByName(name: "Status") {
+                ... on ProjectV2ItemFieldSingleSelectValue {
+                  name
+                }
+              }
+            }
+          }
+        }
+        """
+        variables = {
+            "projectId": PROJECT_ID,
```

Ruff (0.8.2)

Undefined name `PROJECT_ID`

(F821)